applications/crossbar/src/crossbar_auth.erl

```diff
@@ -134,13 +219,7 @@ is_auth_module_enabled(AuthModule, Configs) ->
 %%--------------------------------------------------------------------
 -spec auth_configs(kz_json:object()) -> kz_json:object().
 auth_configs(AccountId) ->
-    MasterId = case kapps_util:get_master_account_id() of
-                   {'ok', Id} -> Id;
-                   {'error', _R} ->
-                       lager:debug("failed to find master account id: ~p", [_R]),
-                       'undefined'
-               end,
-    account_auth_configs(AccountId, MasterId).
+    account_auth_configs(AccountId, master_account_id()).
```

Comment:
All these configs manipulations looks very similar to this: https://github.com/2600hz/kazoo/pull/3312